NF-OSI/dca-template-config.json

```diff
@@ -18,9 +18,9 @@
       {"display_name": "Processed Variant Calls", "schema_name": "ProcessedVariantCallsTemplate", "type": "file"},
       {"display_name": "Processed Expression Data", "schema_name": "ProcessedExpressionTemplate", "type": "file"},
       {"display_name": "FACS Template", "schema_name": "FlowCytometryTemplate", "type": "file"},
-      {"display_name": "Light Scattering Assay", "schema_name": "LightScatteringAssayTemplate", "type": "file"}
+      {"display_name": "Light Scattering Assay", "schema_name": "LightScatteringAssayTemplate", "type": "file"},
+      {"display_name": "Protocol Resource", "schema_name": "ProtocolTemplate", "type": "file"}
     ],
-  "main_fileview" : "syn16858331",
-  "community" : "NF",
-  "schematic_service": "v1"
+  "service_version": "v23.1.1",
```

Comment:
@afwillia, is this the latest version of the service? I pulled this from the Demo config but I see several values across the different projects. Thanks!

Reply:
@allaway the service_version is actually the version of schematic used which is currently 23.9.3. Different projects ran  a [script that generates this file](https://github.com/Sage-Bionetworks/data_curator_config/blob/main/.github/config_schema.py) for various releases of schematic.

Reply:
Cool! I will look into that script in the future.